canonical-web-and-design/cn.ubuntu.com
# Adding Ubuntu 16.04 press release in news section

# Details
- card: https://trello.com/c/PKAeOGRL
## done
- added a new news story based on this [copy doc](https://docs.google.com/spreadsheets/d/1Fm-n0ZLk-YY0-gfYCoes_P1y0tagSepPgxtXXk8vB0M/edit#gid=0)
- got similar images from Yaili's list
## qa
-  go to /news/2016/04/25/ubuntu-1604-released/ and see that it matches the copy doc
- go to /news and see that it's listed under the Ubuntu Kylin section
